lua/lspconfig/server_configurations/ols.lua

```diff
@@ -0,0 +1,20 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'ols' },
+    filetypes = { 'odin' },
+    root_dir = util.root_pattern('ols.json', '.git'),
+    single_file_support = true,
+  },
+  docs = {
+    description = [[
+           https://github.com/DanielGavin/ols
+
+           `Odin Language Server`.
+        ]],
+    default_config = {
+      root_dir = [[util.root_pattern("ols.json", ".git") or current_file_dirname]],
```

Comment:
This is not accurate

Reply:
sorry i copied this from the zls.lua file, 
and did not check with make lint